src/main/webapp/app/course/manage/course-group.component.ts

```diff
@@ -268,4 +274,34 @@ export class CourseGroupComponent implements OnInit, OnDestroy {
         this.rowClass = className;
         setTimeout(() => (this.rowClass = undefined));
     };
+    /**
+     * Method for exporting the csv with the needed data
+     */
+    exportStudentInformation() {
+        if (this.exportReady && this.allCourseGroupUsers.length > 0) {
+            const rows: any[] = this.allCourseGroupUsers.map((user: User) => {
+                const data = {};
+                data[NAME_KEY] = user.name!.trim();
+                data[USERNAME_KEY] = user.login!.trim();
+                data[EMAIL_KEY] = user.email!.trim();
+                return data;
+            });
+            const keys = [NAME_KEY, USERNAME_KEY, EMAIL_KEY];
+            this.exportAsCsv(rows, keys);
+        }
+    }
+    exportAsCsv(rows: any[], keys: string[]) {
```

Comment:
```suggestion
    }

    private exportAsCsv(rows: any[], keys: string[]) {
```
Missing newline and the method does not need to be public. The type of `rows` should potentially be adapted, too, if you want to implement my other suggestion.

Reply:
Solved!